src/Storefront/Resources/views/storefront/element/cms-element-image.html.twig

```diff
@@ -24,7 +24,12 @@
                                         {% endif %}
 
                                         {% sw_thumbnails 'cms-image-thumbnails' with {
-                                            media: element.data.media
+                                            media: element.data.media,
+                                            'xs': '400px',
```

Comment:
Does this actually work? In other areas the `xs`, `sm` and so on are inside `sizes` object like this:

```twig
{# src/Storefront/Resources/views/storefront/component/product/card/box-standard.html.twig #}
{% sw_thumbnails 'product-image-thumbnails' with {
    media: cover,
    sizes: {
        'xs': '501px',
        'sm': '315px',
        'md': '427px',
        'lg': '333px',
        'xl': '284px'
    }
} %}
```

And the standard image sizes are not working? Is there anything legit inside the sizes attribute? Usually default sizes are automatically being set by the `thumbnail.html.twig`:

```
{# src/Storefront/Resources/views/storefront/utilities/thumbnail.html.twig #}

{# set image size for every viewport #}
{% set sizes = {
    'xs': (shopware.theme.breakpoint.sm - 1) ~'px',
    'sm': (shopware.theme.breakpoint.md - 1) ~'px',
    'md': ((shopware.theme.breakpoint.lg - 1) / columns)|round(0, 'ceil') ~'px',
    'lg': ((shopware.theme.breakpoint.xl - 1) / columns)|round(0, 'ceil') ~'px'
} %}
```

Reply:
I could see various sizes created for this in the srcset but I will double check this.

Reply:
@tobiasberge Ah sorry. It was adding the images to the srcset but not actually to the sizes property - https://paste.ec/paste/LMUPpdzM#+JB-24eGl4jySjsyJLcBUz1aR4DXiptgbCA6pX6huxo

I will fix this now.

Reply:
This hasn't been fixed? :-)